english/check-your-phone/check-your-phone.md

```diff
@@ -0,0 +1,12 @@
+Pull out your phone to check the date and see where you are.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.